feature/login/src/main/java/co/kr/tnt/login/LoginScreen.kt

```diff
@@ -51,6 +51,7 @@ import co.kr.tnt.domain.model.AuthType
 import co.kr.tnt.domain.model.LoginResult
 import co.kr.tnt.domain.model.UserType
 import co.kr.tnt.feature.login.R
+import co.kr.tnt.feature.login.R.string.core_next
```

Comment:
```suggestion
import co.kr.tnt.core.ui.R.string.core_next
```
login 모듈에 중복 정의된 것들 제거하고, core 모듈의 문자열 리소스를 참조하도록 수정하면 좋을 것 같습니다 👍